### DISCUSSION TITLE
Exception: Robot returning HTTP error 404

### DISCUSSION THREAD

**Discussion by williamchristianc** [johnwason/abb_motion_program_exec#33] [Q&A]
Hello Everyone,

My name is William and I am new to robot programming.
I have tried to run it in Simulation, which entry point should I choose?
The only one that worked for me is motion_program_init.
After playing the simulation, I could not run the example.py because of this:
Exception: Robot returning HTTP error 404

It will be very helpful, if someone can help me. Thank you so much.

Best regards,
William

**Comment by johnwason** [comment]
Are you using an IRC5 virtual controller? The driver does not support omnicore.

It is possible another program is using port 80. If you go to http://localhost:80 in a browser with robot studio closed, does the browser connect and do you see anything?

**Reply by williamchristianc** [reply]
Hi John, 
Thanks for the reply. Yes, I'm using the IRC5 Virtual Controller.
The browser shows the ISS Windows.

**Comment by johnwason** [comment]
Can you stop iis? The robot controller uses that port for commuication.
